tapalcatl_server/server.go

```diff
@@ -253,15 +253,28 @@ func (bm *OnDemandBufferManager) Get() *bytes.Buffer {
 func (bm *OnDemandBufferManager) Put(buf *bytes.Buffer) {
 }
 
+func logFatalCfgErr(logger JsonLogger, msg string, xs ...interface{}) {
+	logger.Error(LogCategory_ConfigError, msg, xs...)
+	os.Exit(1)
+}
+
 func main() {
 	var listen, healthcheck, debugHost string
 	var poolNumEntries, poolEntrySize int
 	var metricsStatsdAddr, metricsStatsdPrefix string
 
 	hc := handlerConfig{}
 
+	systemLogger := log.New(os.Stdout, "", log.LstdFlags|log.LUTC|log.Lmicroseconds)
+	hostname, err := os.Hostname()
```

Comment:
I debated whether this should be configuration, but ultimately settled on asking the operating system for it, because I think that'll make it easier in the future if we deploy with docker. Can you think of any potential problems here?